wp-content/plugins/duracelltomi-google-tag-manager/integration/whichbrowser/src/Analyser/Header/Useragent.php

```
@@ -0,0 +1,49 @@
+        /* Refine some of the information */
+
+        $this->refineBrowser($header)
+             ->refineOperatingSystem($header);
+    }
+
+    private function removeKnownPrefixes($ua)
+    {
+        $ua = preg_replace('/^OneBrowser\/[0-9.]+\//', '', $ua);
+        $ua = preg_replace('/^MQQBrowser\/[0-9.]+\//', '', $ua);
+        return $ua;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'removeKnownPrefixes'. (undefined)

(UnusedPrivateMethod)